packages/lodestar-params/package.json

```diff
@@ -46,6 +46,8 @@
     "js-yaml": "^3.13.1"
   },
   "devDependencies": {
-    "babel-plugin-inline-import": "^3.0.0"
+    "@types/node-fetch": "^2.5.7",
+    "babel-plugin-inline-import": "^3.0.0",
+    "node-fetch": "^2.6.1"
```

Comment:
maybe better to use axios since we are already using it in lodestar-validator?

Reply:
For blst-ts I was forced to add an HTTP client as dependency to fetch the pre-built bindings. I research what was the least invasive HTTP client and came across this comparison https://github.com/request/request/issues/3143.
- node-fetch has 0 dependencies and is tiny
- axios has 1 dependency and is also quite small
That was the reasoning for picking node-fetch but I would love to know your opinion on which to use given this criteria

Reply:
in blst I would go with pure nodejs and just pipe to file:
```js
const http = require('http');
const fs = require('fs');

const file = fs.createWriteStream("binding.node");
const request = http.get("http://binding.node", function(response) {
  response.pipe(file);
});
```

I would go with axios just because we already have it so we can avoid installing multiple dependencies that are doing same thing. Also it's quite configurable and works on web

Reply:
I did that initially but it broke because this code does not follow redirect and Github assets are not in the original URL

Reply:
I've replaced node-fetch with axios

Reply:
It's easy to fix, works with:
```typescript
const https = require('https');
const fs = require('fs');

const file = fs.createWriteStream("./binding.node");
const request = https.get("<release url>", (response) => {
  if(response.statusCode == 302) {
     https.get(response.headers.location, (res) => {
        res.pipe(file);
     })
  } else {
     response.pipe(file);
  }
});


```